maddymz/irondb
# us-624, t-625,626,630,631

What changed:
- A vanity header has been added to the iron shell to reinforce the tool's name. I want no confusion when communicating with the hand-off team when we say to run the iron shell.
- The argument -x has been retooled to perform a full Docker environment reset. This is done so that we have a powerful tool for troubleshooting the user's machine in case of Docker errors.
- Minor revisions to wording in help. 
- Added some warning suppression so as not to confuse the user with warnings that are harmless.

How to test:
Good news / bad news on that. Good news, if you've been doing any testing that requires running the web app, your virtual drive is probably due for a reset anyway. Bad news, we have to completely remove all of Docker's data and rebuild the server. This is completely safe, no worries.

Checkout the branch and look at the header displayed by the help. Or don't, that part is entirely decoration.
```
git checkout us624
./iron.sh
```

Next we are going to do a full reset on your Docker environment. Docker must be running. Answer yes to both prompts. This will remove all the Docker data stored on your local machine. Afterwards we will rebuild the Docker environment. After it asks for your password, go brew some coffee. It takes about 15 minutes to rebuild the environment.
You can enter each command separately or chain them together.
```
./iron.sh -x
./iron.sh -i
```
-- OR --
```
./iron.sh -xi
```

Open your browser to `localhost:3001` and see that everything is copacetic. Cool? Ok, shutdown the server.
```
./iron.sh -s
```
